.dockerignore

```diff
@@ -0,0 +1 @@
+.git/*
```

Comment:
I assume `.git/` would be better. The difference is that `.git/.foo` would not be matched by `.git/*` (I believe).